### PR TITLE
Package Installer and Charts

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This PR adds the Kubermatic Installer and our Helm charts to the `kubermatic/kubermatic` Docker image and to the archives uploaded to Github. To make usage easier, this PR also provides downloadable archives for Darwin and Windows systems.

In the Docker image, we ship a Helm binary; the Github release archives rely on users having Helm installed on their machine.

**Documentation**:
https://github.com/kubermatic/docs/pull/442

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: hack/ci/ci-github-release.sh [hunk 1]
```diff
@@ -123,73 +149,100 @@ fi
...
+  export GOOS="$(echo "$buildTarget" | cut -d- -f1)"
+  export GOARCH="$(echo "$buildTarget" | cut -d- -f2)"
+
+  echodate "Compiling CE installer ($buildTarget)..."
+  KUBERMATIC_EDITION=ce build_installer
+
+  echodate "Uploading kubermatic CE archive..."
+
+  # switch Docker repository used by the operator to the EE repository
+  yq w -i charts/kubermatic-operator/values.yaml 'kubermaticOperator.image.repository' 'quay.io/kubermatic/kubermatic'
```
Comment (moelsayed) [inline hack/ci/ci-github-release.sh]: I think this is CE.

**Review by moelsayed — COMMENTED** [review]
Tiny nit comment, otherwise lgtm.

File: hack/ci/ci-github-release.sh (hunk 1 above)
Comment (xrstf) [reply hack/ci/ci-github-release.sh]: Will fix soon, good find! :)

Comment (xrstf) [reply hack/ci/ci-github-release.sh]: Fixed.

**Pull request merged**